app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -197,8 +197,18 @@ def handle_telephony_result(method:, default:)
       end
     end
 
-    def track_events
-      analytics.track_event(Analytics::TELEPHONY_OTP_SENT, @telephony_result.to_h)
+    def track_events(otp_delivery_preference:)
+      extra = {
+        area_code: parsed_phone.area_code,
+        country_code: parsed_phone.country_code,
+        phone_fingerprint: Pii::Fingerprinter.fingerprint(parsed_phone.e164),
+        context: context,
+        otp_delivery_preference: otp_delivery_preference,
+        resend: delivery_params[:resend],
+        telephony_response: @telephony_result.to_h,
+        success: @telephony_result.success?,
+      }
+      analytics.telephony_otp_sent(**extra)
```

Comment:
```suggestion
      analytics.telephony_otp_sent(
        area_code: parsed_phone.area_code,
        country_code: parsed_phone.country_code,
        phone_fingerprint: Pii::Fingerprinter.fingerprint(parsed_phone.e164),
        context: context,
        otp_delivery_preference: otp_delivery_preference,
        resend: delivery_params[:resend],
        telephony_response: @telephony_result.to_h,
        success: @telephony_result.success?,
      )
```
I think we can inline this